diff -Naur qwt-6.1.4.org/examples/examples.pro qwt-6.1.4/examples/examples.pro
--- qwt-6.1.4.org/examples/examples.pro	2019-01-02 17:22:32.000000000 +0100
+++ qwt-6.1.4/examples/examples.pro	2021-03-16 21:46:34.206012348 +0100
@@ -49,3 +49,9 @@
         dials \
         controls
 }
+
+#Add extra clean for linker lst files:
+extraclean.commands = find . -type f -name \"*.lst\" -exec rm -v {} \\;
+distclean.depends = extraclean
+QMAKE_EXTRA_TARGETS += distclean extraclean
+
diff -Naur qwt-6.1.4.org/qwtbuild.pri qwt-6.1.4/qwtbuild.pri
--- qwt-6.1.4.org/qwtbuild.pri	2019-01-02 17:22:32.000000000 +0100
+++ qwt-6.1.4/qwtbuild.pri	2021-03-16 21:47:14.158680378 +0100
@@ -57,6 +57,10 @@
     #QMAKE_LFLAGS += -lrt
 }
 
+QMAKE_CFLAGS += -mfpu=neon -v
+QMAKE_CXXFLAGS += -mfpu=neon -v
+QMAKE_LFLAGS += -Xlinker -Map=$@.lst -Xlinker --cref
+
 ######################################################################
 # paths for building qwt
 ######################################################################
diff -Naur qwt-6.1.4.org/qwtconfig.pri qwt-6.1.4/qwtconfig.pri
--- qwt-6.1.4.org/qwtconfig.pri	2019-01-02 17:22:32.000000000 +0100
+++ qwt-6.1.4/qwtconfig.pri	2021-03-16 21:48:52.296297072 +0100
@@ -19,12 +19,12 @@
 QWT_INSTALL_PREFIX = $$[QT_INSTALL_PREFIX]
 
 unix {
-    QWT_INSTALL_PREFIX    = /usr/local/qwt-$$QWT_VERSION
+    #QWT_INSTALL_PREFIX    = /usr/local/qwt-$$QWT_VERSION
     # QWT_INSTALL_PREFIX = /usr/local/qwt-$$QWT_VERSION-qt-$$QT_VERSION
 }
 
 win32 {
-    QWT_INSTALL_PREFIX    = C:/Qwt-$$QWT_VERSION
+    #QWT_INSTALL_PREFIX    = C:/Qwt-$$QWT_VERSION
     # QWT_INSTALL_PREFIX = C:/Qwt-$$QWT_VERSION-qt-$$QT_VERSION
 }
 
@@ -42,14 +42,14 @@
 # runtime environment of designer/creator.
 ######################################################################
 
-QWT_INSTALL_PLUGINS   = $${QWT_INSTALL_PREFIX}/plugins/designer
+#QWT_INSTALL_PLUGINS   = $${QWT_INSTALL_PREFIX}/plugins/designer
 
 # linux distributors often organize the Qt installation
 # their way and QT_INSTALL_PREFIX doesn't offer a good
 # path. Also QT_INSTALL_PREFIX is only one of the default
 # search paths of the designer - not the Qt creator
 
-#QWT_INSTALL_PLUGINS   = $$[QT_INSTALL_PREFIX]/plugins/designer
+QWT_INSTALL_PLUGINS   = $$[QT_INSTALL_PREFIX]/plugins/designer
 
 ######################################################################
 # Features
@@ -63,8 +63,8 @@
 # with every Qt upgrade.
 ######################################################################
 
-QWT_INSTALL_FEATURES  = $${QWT_INSTALL_PREFIX}/features
-# QWT_INSTALL_FEATURES  = $$[QT_INSTALL_PREFIX]/features
+#QWT_INSTALL_FEATURES  = $${QWT_INSTALL_PREFIX}/features
+QWT_INSTALL_FEATURES  = $$[QT_INSTALL_PREFIX]/mkspecs/features
 
 ######################################################################
 # Build the static/shared libraries.
@@ -99,7 +99,7 @@
 # If you want to use a OpenGL plot canvas
 ######################################################################
 
-QWT_CONFIG     += QwtOpenGL
+#QWT_CONFIG     += QwtOpenGL
 
 ######################################################################
 # You can use the MathML renderer of the Qt solutions package to
@@ -110,7 +110,7 @@
 # to your qmake project file.
 ######################################################################
 
-#QWT_CONFIG     += QwtMathML
+QWT_CONFIG     += QwtMathML
 
 ######################################################################
 # If you want to build the Qwt designer plugin,
@@ -171,5 +171,5 @@
 
 unix {
 
-    #QWT_CONFIG     += QwtPkgConfig
+    QWT_CONFIG     += QwtPkgConfig
 }
